src/components/QuranReader/QuranReader.module.scss

```diff
@@ -1,8 +1,12 @@
 @use "src/styles/constants";
 @use "src/styles/breakpoints";
+@use "src/styles/utility";
+
+$quran-reader-padding-mobile: 4rem;
+$quran-reader-padding-tablet: 5rem;
 
 .infiniteScroll {
-  max-width: 95%;
+  @include utility.container();
```

Comment:
nit: does `container` needs `()`?, since it doesn't accept params.
in other utility, such as `mobileM` we don't have `()`. 

I am not super familiar with scss btw, so maybe I am missing something.

Reply:
+1

Reply:
Fixed. Thank you Muhajir!